renderer/vectors/back.js

```diff
@@ -0,0 +1,17 @@
+// Packages
+import React from 'react';
+
+// Components
+import Svg from './svg';
+
+const BackIcon = props => {
+  const svgProps = {...props, viewBox: '0 0 24 24'};
+
+  return (
+    <Svg {...svgProps}>
+      <path d="M20,11V13H8L13.5,18.5L12.08,19.92L4.16,12L12.08,4.08L13.5,5.5L8,11H20Z"/>
+    </Svg>
+  );
+};
+
+export default BackIcon;
```

Comment:
Just export it directly in the `const`.

Reply:
If we `export default` the anonymous function, it doesn't show up with a name in the React DevTools

Reply:
Good point. Wish it would be smarter though and use the name at the callsite.